packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -147,244 +107,499 @@ type TileProps = {
-                    thumbnail.props.className,
-                  ),
-                })
-              ) : (
-                thumbnail
-              )}
-
-              {leftIcon &&
-                React.cloneElement(leftIcon as React.ReactElement, {
-                  className: 'iui-tile-thumbnail-type-indicator',
-                  'data-iui-size': 'small',
-                })}
-
-              {rightIcon &&
-                React.cloneElement(rightIcon as React.ReactElement, {
-                  className: 'iui-tile-thumbnail-quick-action',
-                  'data-iui-size': 'small',
-                })}
-
-              {badge && (
-                <Box className='iui-tile-thumbnail-badge-container'>
-                  {badge}
-                </Box>
-              )}
-            </Box>
-          )}
-
-          <Box className='iui-tile-content'>
-            {variant === 'folder' ? tileName : null}
-
-            {description != undefined && (
-              <Box className='iui-tile-description'>{description}</Box>
-            )}
-
-            {metadata != undefined && (
-              <Box className='iui-tile-metadata'>{metadata}</Box>
-            )}
-
-            {moreOptions && (
-              <DropdownMenu
-                onShow={showMenu}
-                onHide={hideMenu}
-                menuItems={(close) =>
-                  moreOptions.map((option: React.ReactElement) =>
-                    React.cloneElement(option, {
-                      onClick: (value: unknown) => {
-                        close();
-                        option.props.onClick?.(value);
-                      },
-                    }),
-                  )
-                }
-              >
-                <Box
-                  className={cx('iui-tile-more-options', {
-                    'iui-visible': isMenuVisible,
-                  })}
-                >
-                  <IconButton
-                    styleType='borderless'
-                    size='small'
-                    aria-label='More options'
-                  >
-                    <SvgMore />
-                  </IconButton>
-                </Box>
-              </DropdownMenu>
-            )}
-
-            {children}
-          </Box>
-
-          {buttons && <Box className='iui-tile-buttons'>{buttons}</Box>}
-        </Box>
-      </TileContext.Provider>
-    );
-  }) as PolymorphicForwardRefComponent<'div', TileProps>,
-  {
-    /**
-     * Polymorphic Tile action component. Recommended to be used in a "name" of `Tile`.
-     * Renders `a` element by default.
-     * @example
-     * <Tile
-     *   name={<Tile.Action href='/new-page'>Tile name<Tile.Action/>}
-     * />
-     */
-    Action: TileAction,
-  },
-);
+  return (
+    <TileContext.Provider
+      value={{
+        status,
+        variant,
+        isNew,
+        isSelected,
+        isLoading,
+        isActionable,
+        isDisabled,
+        setActionable: () => {},
```

Comment:
I was getting this error
![image](https://github.com/iTwin/iTwinUI/assets/22462084/172e8c0b-b21e-4cd4-80d0-5ed7ca2c3bfc)

Reply:
that's because you removed the code that was passed here before.
![](https://github.com/iTwin/iTwinUI/assets/9084735/887d5a2e-6639-43e9-a4b4-c115a476f46a)

Reply:
ohhh. I moved these logic back.